pkg

```diff
@@ -99,8 +99,8 @@ while getopts :?78abdf:lopqv opt; do
 			PROJECTS=`$SELF -l | sed "s/^/  - /"`
 			<<-HELP_TEXT cat
 			Usage: $SELF [options] [projects]
-			  -7           Build RPMs targeting CentOS 7 (default)
-			  -8           Build RPMs targeting CentOS 8
+			  -7           Build RPMs targeting CentOS 7
+			  -8           Build RPMs targeting CentOS 8 (default)
```

Comment:
Is this true, builds use CentOS8 builds by default?

Reply:
Yes